actors/sunbird-lms-mw/actors/user/src/main/java/org/sunbird/user/actors/KeycloakUserLookupActor.java

```diff
@@ -0,0 +1,59 @@
+package org.sunbird.user.actors;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.sunbird.actor.core.BaseActor;
+import org.sunbird.actor.router.ActorConfig;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.request.Request;
+import org.sunbird.learner.util.UserUtility;
+import org.sunbird.user.dao.UserDao;
+import org.sunbird.user.dao.impl.UserDaoImpl;
+import org.sunbird.user.util.UserLookUp;
+
+@ActorConfig(
+  tasks = {"userSearch"},
+  asyncTasks = {},
+  dispatcher = "most-used-one-dispatcher"
+)
+public class KeycloakUserLookupActor extends BaseActor {
```

Comment:
This actor is not specific to keycloak and so we should have it as UserLookupActor maybe?

Reply:
Currently only keycloak is calling this API, for login and other purpose, so i named it as KeycloakUserLookupActor, And didnt touch private/user/v1/search api as that api is getting used by other team also (Impl team)

Reply:
From a user/org perspective, we have to assume that this API can be used user info by any other service. I still think we should make it generic and not assume that this is only for keycloak. @amit-tarento

Reply:
Commited the chnage , also added validation for lookup type values.